src/attached_probe.cpp

```diff
@@ -68,6 +68,8 @@ bpf_prog_type progtype(ProbeType t)
     case ProbeType::software:   return BPF_PROG_TYPE_PERF_EVENT; break;
     case ProbeType::watchpoint: return BPF_PROG_TYPE_PERF_EVENT; break;
     case ProbeType::hardware:   return BPF_PROG_TYPE_PERF_EVENT; break;
+    case ProbeType::kfunc:      return BPF_PROG_TYPE_TRACING; break;
+    case ProbeType::kretfunc:   return BPF_PROG_TYPE_TRACING; break;
```

Comment:
should we add these to the libbpf.h we vendored, that way we avoid compile issues

Reply:
yep, I noticed that in the CI tests, did not notice out libbpf.h ;-) thnx